src/library/Uno.Themes.Markup.WinUI/Theme.Colors.cs

```diff
@@ -0,0 +1,97 @@
+﻿
+using System;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.MarkupHelpers;
+using Windows.UI;
+
+namespace Uno.Themes.Markup
+{
+	partial class Theme
+	{
+		public static class Colors
+		{
+			public static class Primary
+			{
+				public static Action<IDependencyPropertyBuilder<Color>> Default => ThemeResource.Get<Color>("PrimaryColor");
```

Comment:
Prefer defining an explicit type:
```suggestion

                                delegate IDependencyPropertyBuilder<Color> ColorDependencyPropertyBuilder ();
                                
				public static ColorDependencyPropertyBuilder Default => ThemeResource.Get<Color>("PrimaryColor");
```

Reply:
Ah right sorry forgot about that from the issue feedback